CompetitionTemplate/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -187,4 +190,14 @@ public void setupDashboard(RotateTowardsBallCommand rotate, CollectForwardBallCo
         collect.includeOnSmartDashboard();
         acquire.includeOnSmartDashboard();
     }
+    
+    @Inject
+    public void switchEnableSafeArmOperation(
+            OperatorInterface oi,
+            ArmSubsystem arm,
+            EnableSafeArmOperationCommand enableSafeArm,
+            DisableSafeArmOperationCommand disableSafeArm){
+        oi.operatorPanelButtons.getifAvailable(1).whenPressed(enableSafeArm);
```

Comment:
Actually another option that's worked for us in the past is just have the 1 command like disableSafeArm and have it run whilePressed and then in the end() have it enable safe arm back again. That's probably the best.